frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.module.css

```diff
@@ -40,11 +40,13 @@ body {
     var(--mb-color-text-white) 45%,
     transparent
   );
-  --mb-color-text-selected: var(--mb-base-color-brand-30);
+  --mb-color-text-selected: var(--mb-base-color-white);
+  --mb-color-text-hover: var(--mb-base-color-brand-30);
```

Comment:
previously I was using `*-selected` for hover, so we it'll become `*-hover` now.